ROADMAP.md

```diff
@@ -2,60 +2,47 @@
 
-More detail about short-term delivery plans can be found in the [milestones](https://github.com/vmware-tanzu/community-edition/milestones) in
-the [GitHub project repo](https://github.com/vmware-tanzu/community-edition).
+This roadmap should serve as a reference point for Tanzu Community Edition users to help us prioritize existing features and provide input on unmet needs, and for contributors to understand where the project is heading. Contributors can also consult the roadmap when thinking of proposing new ideas, to determine if their idea conflicts with the roadmap; in that case, the team and community will need to determine whether to adjust the roadmap or to recommend changes to the idea.
 
-## How to help
+Enhancement states help everyone understand if something is in the early stages of an idea or if it has wider community buy-in and committed resources. This is important because it helps limit waste, and focus work on the most important and timely activities. The stages are described below:
 
-Discussion on the roadmap can take place in threads under [Issues](https://github.com/vmware-tanzu/community-edition/issues)
-or in [community meetings, our Google Group, or our Slack channel](https://tanzucommunityedition.io/community/).
-Please open and comment on an issue if you want to provide suggestions and feedback to an item in the roadmap.
-Please review the roadmap to avoid potential duplicated effort.
+* Awaiting Proposal - these enhancements are in the idea mode, and the project maintainers are awaiting a detailed proposal. This state is useful to explore different solutions to a known problem and encourage discussion.
+* Proposal in Review - maintainers are currently reviewing the proposal, which includes resource requirements and commitments.
+* Implementation in Progress - engineering work has begun and will be tracking towards a release target.
 
-## Need an idea for a contribution?
+## How to help
 
-Check out our [issues](https://github.com/vmware-tanzu/community-edition/issues) for issues tagged
-[help-wanted](https://github.com/vmware-tanzu/community-edition/issues?q=is%3Aopen+is%3Aissue+label%3Ahelp-wanted) or
-[good-first-issue](https://github.com/vmware-tanzu/community-edition/labels/good-first-issue).
+Discussion on the roadmap can take place in threads under Issues or in community meetings, our Google Group, or our Slack channel. Please open and comment on an issue if you want to provide suggestions for feedback on  an item in the roadmap.  Community members are encouraged to be actively involved, and also stay informed so contributions can be made with the most positive effect and limited duplication of effort.
```

Comment:
> Discussion on the roadmap can take place in threads under Issues or in community meetings, our Google Group, or our Slack channel

This seems really ambiguous to me. If I'm a new contributor, or even an existing contributor, it's overwhelming to be expected to track and communicate within so many different avenues without clear direction.

Should we make a recommendation on the _preferred_ communication channel?

i.e.

```diff
- Discussion on the roadmap can take place in threads under Issues or in community meetings, our Google Group, or our Slack channel
+ Discussion on the roadmap should take place in threads under Issues. Although we welcome discussion in community meetings, our Google Group, or our Slack channel.
```

Reply:
Updated.